src/styles.css

```diff
@@ -0,0 +1,38 @@
+h1 {
```

Comment:
I am not sure sure about mixing tailwind with classic CSS, but I haven't been apart of any IDEM discussions so I will leave that up to you

Reply:
I am going to change this in my next PR, but just want to get it in there for now. :)